src/Denormalizer/Denormalizer.php

```diff
@@ -219,4 +228,31 @@ private function getSubPathsByNames(string $path, array $names): array
 
         return $subPaths;
     }
+
+    /**
+     * @param DenormalizerContextInterface $context
+     * @param object                       $object
+     * @param array                        $missingFields
+     */
+    private function resetMissingFields(DenormalizerContextInterface $context, $object, array $missingFields)
```

Comment:
what is the purpose of the whole "resetMissingField" functionality?

Reply:
PUT calls, if a field is missing you should get the original state (temporary its null, but i will adapt with the factory reading the value of a newly created object

Reply:
ok i get it now, so in the put action i set the resetMissingFields in the context but not in the patch action, right?

Reply:
@healerz yes